afs/parsers.py

```diff
@@ -109,9 +111,12 @@ def manifest_parser(notebook_path, pypi_endpoint, output_dir=None, manifest_yaml
         requirements_append = [req for req in manifest['requirements'] if req not in req_list]
 
         if afs_sdk_version:
-            for afs_req in [req for req in requirements_append if req.startswith('afs')]:
+            add_new_version = False
+            for afs_req in [req for req in requirements_append if req=='afs']:
```

Comment:
Is there will be multiple `afs_req`? If so, maybe we should consider to remove duplicate `req`.

Reply:
Normally, there is only version in user requirement. If afs is not specific version in user requirement, it will be removed and assign the new version.

Reply:
I mean, you are using `for loop` right here, and that's mean there might be multiple `req` fit this `if req='afs'` statment and return as a list.
For example:
```python
manifest = {
    'requirements': [
        'afs',
        'afs'
    ]
}
```
Current implement looks like it can handle this example. But, maybe we should remove all duplicated `req` before we start to use the `requirements_append` variable rather than only handle about `afs`. Any thought?

Reply:
Maybe we can use `set` to handle duplicate package?

Reply:
Right.